src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -282,6 +282,9 @@
   <data name="Introduce_local_constant_for_all_occurrences_of_0" xml:space="preserve">
     <value>Introduce local constant for all occurrences of '{0}'</value>
   </data>
+  <data name="Introduce_local_for_all_occurrences_of_0_including_writes" xml:space="preserve">
+    <value>Introduce local for all occurrences of '{0}' including writes</value>
```

Comment:
Can we change the original text to be `Introduce local for all reads of '{0}'.` and this description to `Introduce local for all writes and reads of '{0}'.`

Reply:
I much prefer your names, done!